packages/editor-ui/src/views/WorkflowOnboardingView.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+import { VIEWS } from '@/constants';
+import { useTemplatesStore, useWorkflowsStore } from '@/stores';
+import { onMounted } from 'vue';
+import { useRoute, useRouter } from 'vue-router';
+
+const templateStore = useTemplatesStore();
+const workfowStore = useWorkflowsStore();
+const router = useRouter();
+const route = useRoute();
+
+const openWorkflowTemplate = async (templateId: string) => {
+	try {
+		const template = await templateStore.getFixedWorkflowTemplate(templateId);
+		if (!template) {
+			throw new Error();
```

Comment:
Should we add a description to the error so we can see it on Sentry

Reply:
hmm it's getting caught so won't show up on sentry.. interesting, ya I guess we can add alerting for this.